### ISSUE TITLE
AppCenter publishing?

### ISSUE THREAD

**Issue by Hedronmx** [jeremypw/bulk-renamer#1]
Are you publishing this app?

**Comment by jeremypw** [comment]
Yes, when I am happy it has reached a minimum standard it will be published in the elementaryos AppCenter (if accepted).

**Comment by jeremypw** [comment]
The app has now been submitted to AppCenter and is awaiting human review.
